lib/components/font_style.dart

```diff
@@ -1,9 +1,32 @@
-enum FontStyle {
-  bold(1),
-  italic(3),
-  underline(4),
-  strikethrough(9);
-
+class FontStyle {
   final int code;
-  const FontStyle(this.code);
+  final String name;
+
+  const FontStyle._(this.code, this.name);
+
+  static const bold = FontStyle._(1, 'bold');
+  static const italic = FontStyle._(3, 'italic');
+  static const underline = FontStyle._(4, 'underline');
+  static const strikethrough = FontStyle._(9, 'strikethrough');
+
+  static const mapping = <int, FontStyle>{
+    1: bold,
```

Comment:
If you don't want to duplicate the mapping. You could consider making it:

```dart
static final _fromCode = {
  for (final style in values)
    style.code: style,
}
```

Less efficient at runtime, but more maintainable because there is no duplication. (Same for `Color`.)

I'm fine with both options.